dotcom-rendering/src/web/components/ImageComponent.tsx

```diff
@@ -319,13 +346,22 @@ export const ImageComponent = ({
 				{!!title && (
 					<ImageTitle title={title} role={role} palette={palette} />
 				)}
+				<Island deferUntil="interaction">
+					<LightboxButton
+						role={role}
+						format={format}
+						elementId={element.elementId}
+						isMainMedia={isMainMedia}
+					/>
+				</Island>
```

Comment:
Should we be checking if the image supports lightbox (e.g the >620px check) before adding the button?

Reply:
We should.

We [are now](https://github.com/guardian/dotcom-rendering/pull/7129/commits/826ada4cb85010bdb29f734e0a6bc86810f50015).